src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -55,10 +55,6 @@ public CommandResult execute(Model model) throws CommandException {
                 ? getRecipe(lastShownList, toDeleteIndex)
                 : getRecipe(lastShownList, toDeleteName);
```

Comment:
I was worried that this ternary operator might introduce a bug since it does not consider if both `toDeleteIndex` and `toDeleteName`, but it looks like that would be handled by their `getRecipe`. Good job! 👍